src/index.js

```diff
@@ -1,32 +1,76 @@
 /* eslint-disable no-unused-vars */
+/* eslint-disable prefer-destructuring */
+
 import _ from 'lodash';
 import './style.css';
+import bin from './bin.svg';
+
+import { addNewComponent, removeComponents } from './modules/functions.js';
 
-const components = [
-  {
-    description: 'complete to Do list project',
-    completed: false,
-    index: 1,
-  },
-  {
-    description: 'wash the dishes',
-    completed: false,
-    index: 0,
-  },
-];
-
-const showComponents = () => {
+const showComponents = (components) => {
   const container = document.querySelector('.to-do-list');
+  container.innerHTML = ''; // Clears the container
 
   // Sort components array based on index property
   components.sort((a, b) => a.index - b.index);
 
   components.forEach((component) => {
     const items = document.createElement('li');
     items.className = 'list-item';
-    items.innerHTML = `<input class="check-box" type="checkbox">${component.description}`;
+    items.dataset.index = component.index;
+    items.innerHTML = `
+    <div class="listboxflex">
+        <input class="check-box" type="checkbox">
+        <input class="description" value="${component.description}" readonly>
+        <a class="delete-icon"><img class="delete-img" src="${bin}" alt="bin"></a>
+    </div>
```

Comment:
- [ ] I discovered that this image is broken, I was able to access the functionality through the `alt` attribute. Kindly fix it. 👍 

<img width="320" alt="Untitled" src="https://github.com/Bupilipili/To-do-list/assets/103386183/f245b8d4-e162-447b-bd1f-3deb5d8f154b">

Reply:
the image is visible